symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientBuilderWebClient.java

```diff
@@ -207,6 +209,15 @@ protected SslContext createSSLContext() {
           keyManagerFactory.init(keyStore, this.keyStorePassword.toCharArray());
           builder.keyManager(keyManagerFactory);
         }
+
+        // if logging debug is enabled, we print the truststore entries
```

Comment:
and if we can factor this if we make it a method and move it to the http-api module